EventEmitter.js

```diff
@@ -130,10 +141,17 @@
      * @return {Object} Current instance of EventEmitter for chaining.
      */
     proto.addListener = function addListener(evt, listener) {
+
+        if (listener === null) { return this; }
```

Comment:
If you're checking for `null` + `undefined` this should be `==`, not `===`. Not sure I like the silent failure either. I'd rather it let you add null or throw a type error.

Just checked: Node throws a type error.

> TypeError: listener must be a function